packages/legend-graph/src/graph/metamodel/pure/packageableElements/service/PostValidation.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { hashArray, type Hashable } from '@finos/legend-shared';
+import { CORE_HASH_STRUCTURE } from '../../../../../graph/Core_HashUtils.js';
+import type { RawLambda } from '../../rawValueSpecification/RawLambda.js';
+import type { PostValidationAssertion } from './PostValidationAssertion.js';
+
+export class PostValidation implements Hashable {
+  description!: string;
+  parameters: RawLambda[] = [];
+  assertions: PostValidationAssertion[] = [];
+
+  get hashCode(): string {
+    return hashArray([
+      CORE_HASH_STRUCTURE.SERVICE_PARAMETER_VALUE,
```

Comment:
Please change this to `CORE_HASH_STRUCTURE.SERVICE_POST_VALIDATION`

Reply:
opps! missed this! thanks!